schema/Query.graphql

```diff
@@ -28,4 +28,14 @@ type Query {
     Generate challenge to authenticate with WebAuthn
     """
     generateAuthenticatorChallenge(username: String!): AuthenticationWithWebPublicKeyCredential
+
+    """
+    Get a life by its ID
+    """
+    getLife(id: ObjectID!): Life!
+
+    """
+    List lives
+    """
+    listLives: [Life]!
```

Comment:
For your list, suggest it to be`[Life!]!` as you would not want null values returned in your array